packages/plugin-ext/src/main/browser/scm-main.ts

```diff
@@ -258,6 +262,15 @@ export class PluginScmProvider implements ScmProvider {
         this.onDidChangeResourcesEmitter.fire();
     }
 
+    toUrl(uri: vscodeURI | undefined): string | undefined {
+        if (!uri) { return undefined; }
+        if (!uri.path) { return undefined; }
+
+        return `url('${new Endpoint({
+            path: `hostedPlugin${uri.path}`
+        }).getRestUrl().toString()}')`;
```

Comment:
minor: we can format this to one line, I think it's clear enough

```suggestion
        return `url('${new Endpoint({ path: `hostedPlugin${uri.path}` }).getRestUrl().toString()}')`;
```

Reply:
Okay. Done